README.md

```diff
@@ -514,74 +321,53 @@ sudo systemd-sysext refresh
+3. If applicable, create service unit(s) for the application in `EXTNAME/usr/lib/systemd/system/`.
+4. Call `bake.sh EXTNAME` to generate `EXTNAME.raw` from the subdirectory we populated in steps 1. to 3.
+   A number of environment variables control metadata generated by `bake.sh`
+   - `ARCH` - CPU architecture. Either `arm64` or `x86-64`.
+   - `RELOAD` - set this to "1" if your sysext includes service units (step 3. above).
+     This will instruct the service manager to reload its configuration after merge, so the new service files will be picked up.
 
-See the above intro section on how to use the resulting sysext image.
+- Add the extension build script to the repo , add a corresponding markdown file with usage instructions and config snippets to `docs/`.
+- Optionally add the new extension name to [release_build_versions.txt](https://github.com/flatcar/sysext-bakery/blob/main/release_build_versions.txt) to automatically build sysexts with new bakery releases.
+- Add the sysext to the list of available extensions in this README.
+- File a PR.
+Done!
 
-You can also limit the sysext image to only Docker (without containerd and runc) or only containerd (no Docker but runc) by passing the environment variables `ONLY_DOCKER=1` or `ONLY_CONTAINERD=1`.
-If you build both sysext images that way, you can load both combined and, e.g., only load the Docker sysext image for debugging while using the containerd sysext image by default for Kubernetes.
 
-### Baking sysexts into Flatcar OS images
-
-Using the `bake_flatcar_image.sh` script, custom Flatcar OS images can be created which include one or more sysexts.
-The script will download a Flatcar OS release image, insert the desired sysexts, and optionally create a vendor (public / private cloud or bare metal) image.
-
-By default, the script operates with local sysexts (and optionally sysupdate configurations if present).
-However, the `--fetch` option may be specified to fetch the sysext `.raw` file and sysupdate config from the latest Bakery release.
-
-Sysexts can be added to the root partition or the OEM partition of the OS image (root is preferred).
-Read more about Flatcar's OS image disk layout here: https://www.flatcar.org/docs/latest/reference/developer-guides/sdk-disk-partitions/
-
-The script requires sudo access at certain points to manage loopback mounts for the OS image partitions and will then prompt for a password.
-
-Refer to `./bake_flatcar_image.sh --help` for more information.
-
-Example usage:
-```
-./bake_flatcar_image.sh --fetch --vendor qemu_uefi wasmtime:wasmtime-24.0.0-x86-64.raw
-```
-
-Example usage with local sysext:
-```
-ls -1
-  myext-1.0.1-x86-64.raw
-  myext.conf
-./bake_flatcar_image.sh --fetch --vendor qemu_uefi myext:myext-1.0.1-x86-64.raw
-```
-
-The script supports all vendors and clouds natively supported by Flatcar.
-
-### Flix and Flatwrap
+## Flix and Flatwrap
 
 The Flix and Flatwrap tools both convert a given chroot folder into a systemd-sysext image.
```

Comment:
It's not entirely clear to me what "chroot folder" implies. I presume it's a folder that I would chroot into (a chroot-able folder?) If that's the case, the above instructions indicate that only files ina `/usr` hiararchy are considered valid. this would be a contradiction?

Reply:
It's actually just "root" folder; updated.